### PR TITLE
[dist] support fetching release-lines from warehouse.ai

### PR DESCRIPTION
**Pull request by jcrugzz**

### CODE REVIEW

File: lib/routes/release-line.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const a = require('express-async-handler');
```
Comment (msluther) [inline lib/routes/release-line.js]: a?

File: test/integration/routes/release-line.test.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      body = await request({
+        json: true,
+        uri: address(app, { pathname: 'release-line/pkg/1.0.0' })
+      });
+      assume(body.pkg).equals(pkg);
+      assume(body.version).equals(version);
+      assume(body.dependents).is.an('object');
+      assume(Object.keys(body.dependents)).has.length(0);
+    } catch (ex) {
+      throw ex;
```
Comment (msluther) [inline test/integration/routes/release-line.test.js]: leave off the catch? you can just do try...finally right?

File: lib/routes/release-line.js (hunk 1 above)
Comment (fritzmonkey) [inline lib/routes/release-line.js]: I think we should name this something with more letters so it is easier to find when scanning, but I also can't think of anything meaningful.

Comment (msluther) [reply lib/routes/release-line.js]: `asyncIt`, `makeItAsync`, `asyncify`, `doItAsync`, `asunk`

File: test/integration/routes/release-line.test.js (hunk 2 above)
Comment (jcrugzz) [reply test/integration/routes/release-line.test.js]: @msluther i wasnt sure if it auto threw. i can do that if it does

File: lib/routes/release-line.js (hunk 1 above)
Comment (jcrugzz) [reply lib/routes/release-line.js]: @msluther @fritzmonkey short for async basically. i just didnt like `asyncHandler` since its so long. Ideas?

Comment (jcrugzz) [reply lib/routes/release-line.js]: @fritzmonkey @msluther what about `asuck`

File: package-lock.json [hunk 3]
```diff
@@ -86,7 +137,7 @@
     },
     "accepts": {
       "version": "1.3.5",
-      "resolved": "https://artifactory.secureserver.net/artifactory/api/npm/node-virt/accepts/-/accepts-1.3.5.tgz",
+      "resolved": "https://artifactory.secureserver.net/artifactory/api/npm/node-virt/accepts/-/accepts-1.3.5.tgz?dl=https://registry.npmjs.org/accepts/-/accepts-1.3.5.tgz",
```
Comment (SivanMehta) [inline package-lock.json]: This should not have been merged

**Pull request merged**